docs/lesiaturuta/PokemonProject/style.css

```diff
@@ -0,0 +1,776 @@
+}
+
+/*CHECKBOX*/
+
+.login__form-checkbox-input,
+.form__search-radio-radio,
+.form__search-checkbox-checkbox {
+  position: absolute;
+  z-index: -1;
+  opacity: 0;
+}
+
+.login__form-checkbox-input,
+.login__form-checkbox-name,
+.form__search-radio-radio,
+.form__search-checkbox-checkbox,
+.form__search-radio-label,
+.form__search-checkbox-label {
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+}
+
+.form__search-radio-radio::before,
+.form__search-radio-label::before {
+  content: "";
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  flex-shrink: 0;
+  flex-grow: 0;
+  border: 1px solid #FFF;
+  border-radius: 50%; /* Используйте 50% для создания круга */
+  margin-right: .5em;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 50% 50%;
+}
+
+.login__form-checkbox-input ::before,
+.login__form-checkbox-name::before {
+  content: "";
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  flex-shrink: 0;
+  flex-grow: 0;
+  border: 1px solid #EF4934;
+  border-radius: .25em;
+  margin-right: .5em;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 50% 50%;
+}
+
+.form__search-checkbox-checkbox::before,
+.form__search-checkbox-label::before {
+  content: "";
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  flex-shrink: 0;
+  flex-grow: 0;
+  border: 1px solid #FFF;
+  border-radius: .25em;
+  margin-right: .5em;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 50% 50%;
+}
+
+.login__form-checkbox-input:checked + .login__form-checkbox-name::before,
+.form__search-radio-radio:checked + .form__search-radio-label::before,
+.form__search-checkbox-checkbox:checked + .form__search-checkbox-label::before {
+  border-color: #0B76EF;
+  background-color: #0B76EF;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3e%3cpath fill='%23fff' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z'/%3e%3c/svg%3e");
+}
+
+/*FOOTER*/
+
+.footer {
+  margin-top: 20px;
+  width: 100%;
+  min-height: 160px;
+  background: #EF4934;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 10px;
+}
+
+.footer__icons {
+  display: flex;
+}
+
+.footer__description,
+.footer__description-link {
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+  color: #FFF;
+}
+
+.footer__description-link {
+  text-decoration: none;
+}
+
+.item {
+  list-style-type: none;
+}
+
+.item__link-facebook,
+.item__icon-linkedin,
+.item__icon-instagram {
+  cursor: not-allowed;
+}
+
+.footer__description {
+  margin-top: 20px;
+  text-align: center;
+}
+
+/*LOGIN*/
+
+.login {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex: 1;
+}
+
+.login__form {
+  max-width: 575px;
+  width: 100%;
+  height: 520px;
```

Comment:
Please remove fixed height and use paddings or gaps for the alignment elements

Reply:
fixed